packages/mermaid/src/diagrams/flowchart/parser/flow.jison

```diff
@@ -567,9 +573,9 @@ direction
     { $$={stmt:'dir', value:'LR'};}
     ;
 
-alphaNumToken  : PUNCTUATION | AMP | UNICODE_TEXT | NUM| ALPHA | COLON | COMMA | PLUS | EQUALS | MULT | DOT | BRKT| UNDERSCORE ;
+alphaNumToken  : PUNCTUATION | AMP | UNICODE_TEXT | unicodeTitleStatement | NUM| ALPHA | COLON | COMMA | PLUS | EQUALS | MULT | DOT | BRKT| UNDERSCORE ;
 
-idStringToken  : ALPHA|UNDERSCORE |UNICODE_TEXT | NUM|  COLON | COMMA | PLUS | MINUS | DOWN |EQUALS | MULT | BRKT | DOT | PUNCTUATION | AMP | DEFAULT;
+idStringToken  : ALPHA|UNDERSCORE |UNICODE_TEXT | unicodeTitleStatement | NUM|  COLON | COMMA | PLUS | MINUS | DOWN |EQUALS | MULT | BRKT | DOT | PUNCTUATION | AMP | DEFAULT;
```

Comment:
`unicodeTitleStatement` casing should be consistent with the rest of the tokens.

Reply:
Moving the logic into Diagram.ts will make the jison changes unnecessary.